### ISSUE TITLE
Bug in DevTools

### ISSUE THREAD

**Issue by Chyyeesh** [GoogleChrome/developer.chrome.com#4328]
Good afternoon! My name is Nadya
I am engaged in programming and faced the problem of displaying data in the site code.
I'm writing code in Visual Studio Code so that the border for the box is 3 pixels, but when I look at the code on the page, it shows a smaller size. My friends have a similar code when I asked them to open it and everything is fine with their sizes. Shows exactly 3 pixels.
The scale also checked everything is fine. Such a problem is up to 5 pixels, that is, when I write 1, 2, 3, 4 - it shows a smaller number, as soon as I write exactly 5, then everything is fine. I'll attach screenshots below. Tell me what can be done about it, please.
![vs1](https://user-images.githubusercontent.com/105556935/202448540-8e3ce3ae-3cd7-44e0-8c60-52d0210e8052.png)
![vs2](https://user-images.githubusercontent.com/105556935/202448544-460720d9-362f-4eba-95b5-f05295a8de91.png)
![vs3](https://user-images.githubusercontent.com/105556935/202448548-0897e7e2-18fd-403c-9fd1-7baf97f5434c.png)
![vs4](https://user-images.githubusercontent.com/105556935/202448552-8dd691a4-73b2-4c81-ba14-1e022a42244b.png)
![vs5](https://user-images.githubusercontent.com/105556935/202448555-524ac366-cb54-4d81-a26c-ef693c8d5241.png)

**Comment by AaronForinton** [comment]
@sofiayem Is this bug related to any of our docs?

**Comment by sofiayem** [comment]
@AaronForinton Most probably that's not doc-related. But I've got this.

@Chyyeesh Thank you for reaching out. Unfortunately, this repository is for [DevTools documentation](https://developer.chrome.com/docs/devtools/), not DevTools itself. However, might I suggest a several things to do:

1. Open the Elements > Computed tab (next to Styles), enable the "Show all" checkbox, filter for, for example, `border-top-width` property, and check the value and, more importantly, the value's unit. Is it pixels? The value `.995` looks like it might be a conversion rounding error.
2. Third-party software, such as some kind of Chrome extension, might be interfering with scaling and/or styling. Try disabling extensions.
3. If you still can't figure out what's going on, feel free to file a bug at [crbug.com](crbug.com). This is the bug tracker for everything Chrome. (If I'm not wrong, anyone can file a bug there.)

Closing this issue now as it's not documentation-related.

P.S. Счастливой веб-разработки! :)
